samples/vsphere-disk-manager.py

```diff
@@ -0,0 +1,309 @@
+#!/usr/bin/env python
+
+"""
+Written by Miquel Adrover
+https://github.com/madrover
+
+Forked and extended from https://github.com/vmware/pyvmomi-community-samples/blob/master/samples/add_disk_to_vm.py # nopep8
+
+Script to manage independent disks on vSphere.
+
+The following disk operations are available:
+- add
+- attach
+- detach
+- list
+- destroy
+
+Known issues:
+This will not add more than 15 disks to a VM
+To do that the VM needs an additional scsi controller
+and I have not yet worked through that
+"""
+
+from pyVmomi import vim
+from pyVim.connect import SmartConnect, Disconnect
+import atexit
+import argparse
+import getpass
+import ssl
+import sys
+import time
+
+
+def get_args():
+    parser = argparse.ArgumentParser(
+        description='Disk manager for vSphere')
+
+    parser.add_argument('--operation',
+                        required=True,
+                        action='store',
+                        choices=['add', 'attach', 'detach', 'list', 'destroy'],
+                        default='add',
+                        help='Operation to execute')
+
+    parser.add_argument('-s', '--host',
+                        required=True,
+                        action='store',
+                        help='vSphere server')
+    parser.add_argument('-o', '--port',
+                        type=int,
+                        default=443,
+                        action='store',
+                        help='vSphere port')
+
+    parser.add_argument('-u', '--user',
+                        required=True,
+                        action='store',
+                        help='Username')
+
+    parser.add_argument('-p', '--password',
+                        required=False,
+                        action='store',
+                        help='Password')
+
+    parser.add_argument('-v', '--vm-name',
+                        required=False,
+                        action='store',
+                        help='name of the vm')
+
+    parser.add_argument('--uuid',
```

Comment:
First, Sorry for delaying reviewing this.

how about making uuid or vm-name any one of them as optional. If I look at the code both are used to find VM.